job-dsls/jobs/deploy_jobs.groovy

```diff
@@ -158,12 +158,17 @@ def final REPO_CONFIGS = [
                 ],
                 downstreamRepos        : []
         ],
-        // following repos are not in repository-list.txt, but we want a deploy jobs for them
         "optaweb-employee-rostering" : [
                 ircNotificationChannels: ["#optaplanner-dev"],
                 artifactsToArchive     : DEFAULTS["artifactsToArchive"] + [
                         "**/target/configurations/cargo-profile/profile-log.txt"
                 ]
+        ],
+        "optaweb-vehicle-routing" : [
+                ircNotificationChannels: ["#optaplanner-dev"],
+                artifactsToArchive     : DEFAULTS["artifactsToArchive"] + [
+                        "**/target/configurations/cargo-profile/profile-log.txt"
```

Comment:
Nope. Cargo is not used in this repository. Let's keep just defaults.

Reply:
removed